feedback-front/static/js/histogram.js

```diff
@@ -0,0 +1,21 @@
+function render_bar_chart(data){
+  nv.addGraph(function() {
+    var chart = nv.models.discreteBarChart()
+                .x(function(d) { return d.label })
+                .y(function(d) { return d.value })
+                .staggerLabels(true)
+                .tooltips(false)
+                .showValues(true)
+
+    d3.select('#chart svg')
```

Comment:
what if I want to render in another div with a different div? The '#chart' part of this should be configurable